gtk3/lib/gtk3/tree-iter.rb

```diff
@@ -73,6 +73,10 @@ def children
       @model.iter_children(self)
     end
 
+    def first_child
+      @model.iter_nth_child(self, 0)
```

Comment:
Yes of course I will but I have one question, I thought (or at least I had the feeling) that how I did it was more efficient, look:

```
def first_child
   @model.iter_nth_child(self, 0)
end
```

*    ruby get the instance variable value
*    then ask its class `Gtk::TreeStore` for the method `#iter_nth_child` and so on.

And for me the usage of the `#nth_child` method we have defined before, add another steps : 

```
def first_child
    nth_child(0)
end
```
*   get self class, 
*   ask for the method nth_child
*    ruby get the instance variable value
*    then ask its class `Gtk::TreeStore` for the method `#iter_nth_child` and so on.

I well understand that this example taken alone is not sufficient to assure a better efficiency of the code but I though that in complexe code the cumulated efficiency of the code could matter.

I have a naive/basic knowledge of ruby and maybe it must have some magic code behind the ruby interpreter that make that the two previous codes are the same.

Or maybe you think that the readability if more important thant the efficiency we could add?

As I have said before I will do what you asked me to do, I just try to have a better understanding of what I am doing.